tests/unit/test_additionalfilescache.py

```diff
@@ -14,20 +14,14 @@
 
 def test_binaries_and_datas():
     datas, binaries, modnames = [
-        [('source', 'dest'), ('src', 'dst')],
-        [('abc', 'def'), ('ghi', 'jkl')],
-        ['testmodule1', 'testmodule2']
+        [('source', 'dest'), ('src', 'dst')], [('abc', 'def'), ('ghi', 'jkl')], ['testmodule1', 'testmodule2']
```

Comment:
These should be simply assigned separately to each variable. There's no reason to construct a list and then deconstruct it into three variables...